dune_client/base_client.py

```diff
@@ -35,7 +39,11 @@ def from_env(cls) -> BaseDuneClient:
         without having to import dotenv or os manually
         We use `DUNE_API_KEY` as the environment variable that holds the API key.
         """
-        return cls(os.environ["DUNE_API_KEY"])
+        return cls(
+            api_key=os.environ["DUNE_API_KEY"],
+            base_url=os.environ.get("BASE_API_URL", "https://api.dune.com"),
+            request_timeout=float(os.environ.get("REQUEST_TIMEOUT", 10)),
```

Comment:
Should the environment variables be prefixed with `DUNE_API` to "namespace" them?

Reply:
Sure!